### ISSUE TITLE
[bug] Vanced microG high battery drain on Redmi Note 8 Pro

### ISSUE THREAD

**Issue by jastreb** [YTVanced/VancedMicroG#162]
I am not sure how to prove this other than stating the obvious, that after I removed all Vanced apps battery drain stopped.

Yes microG and YT Vanced were excluded from battery optimisation.
Yes microG was set to autostart.
MIUI version is 12.5.3 and I believe drain started after last update, not sure if update of MIUI or Vanced
Device is a year old. I am not using mobile excessively so battery used to last up to 3 days, problem was noticed after it dropped to less than a day. It was so severe that I thought battery or device has gone bad, hence started seeking for a solution.
It appears that issue is fixed after I removed Vanced apps. Please advise since I dont know how to reproduce or add more helpful info. Logs or whatever is needed.

**Comment by vfonic** [comment]
Same here. :/

This is the screenshot from battery usage since last night (note that it's 7:15am in the morning on the screenshot):
![Screenshot_20211008-071626](https://user-images.githubusercontent.com/67437/136502410-903bb6b6-7d5d-41ab-9f50-53b2064ce2b2.jpg)

I went to sleep with about 80% battery.

**Comment by Srdjan667** [comment]
It happened to me too. Last night, I had 75%. In the morning I had 6%. When I go to battery usage it says it's Google Play Services, but microG is specifically designed to work with Google Play Services.
I have Xiaomi Redmi Note 8 running Android 9 amd MIUI 11.
![Screenshot_2021-10-08-07-50-25-646_com miui securitycenter](https://user-images.githubusercontent.com/73754173/136505930-0981c833-634f-4fa6-8384-40237bd67d18.jpg)

**Comment by Jastreb** [comment]
> 
> 
> Same here. :/
> This is the screenshot from battery usage since last night (note that it's 7:15am in the morning on the screenshot)
> 
> I went to sleep with about 80% battery.
> 
> EDIT: This was on OnePlus 6, OxygenOS 11.1.1.1 (Android 11)



> 
> 
> It happened to me too. Last night, I had 75%. In the morning I had 6%. When I go to battery usage it says it's Google Play Services, but microG is specifically designed to work with Google Play Services. I have Xiaomi Redmi Note 8 running Android 9 amd MIUI 11. ![Screenshot_2021-10-08-07-50-25-646_com miui securitycenter](https://user-images.githubusercontent.com/73754173/136505930-0981c833-634f-4fa6-8384-40237bd67d18.jpg)

Yes very strange, so its not just MIUI. I recommend you guys try to remove all vanced apps and see if issue persists. If not then its obviously vanced issue.

**Comment by gghhkm** [comment]
[Settings Up Adb](https://github.com/gghhkm/Vanced-Issue-Center/blob/main/Guides/SettingUpAdb.md)
[Creating A Logcat](https://github.com/gghhkm/Vanced-Issue-Center/blob/main/Guides/CreatingALogcat.md)

**Comment by abdelruhman178** [comment]
![Screenshot_2023-04-25-03-54-47-990_com miui securitycenter](https://user-images.githubusercontent.com/131730806/234156842-cd9c2e58-cc90-40f9-ae45-9667bb273d63.jpg)
 Is this normal

**Comment by rothschild86** [comment]
Still an issue on note 8 pro in 2024. Massive drain that stopped after removing microG...
